### PR TITLE
Update AlligatorsTooth.java

### PR DESCRIPTION
**Pull request by RealSoy**
## Describe your changes
changed alligator tooth sound and effect to mineplex's (it's objectively better)

## Link to issue (if applicable)
https://streamable.com/0dvhfp

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested my changes.

### CODE REVIEW

**Comment by ezrn** [comment]
I like the current one way more

**Comment by RealSoy** [comment]
you might be deaf sir!

**Comment by ezrn** [comment]
> you might be deaf sir!

you cant call it objectively better and have yours sound like 50 sheep drowning in water instead of the smooth bubble sound it has now

**Comment by RealSoy** [comment]
"sound like 50 sheep drowning in water" bro what are you saying

the new one sounds like when i piss onto water and there is no reason for the lapis effect to play every single nanosecond

**Comment by ezrn** [comment]
> "sound like 50 sheep drowning in water" bro what are you saying
> 
> the new one sounds like when i piss onto water and there is no reason for the lapis effect to play every single nanosecond

lapis blocks can be played less frequently, but the bubble sound is much nicer

**Comment by RealSoy** [comment]
it just sounds weird, no one complained about the old sound why change it. same thing with windblade THE NEW SOUND IS SO AWFUL LOL

**Comment by Mykindos** [comment]
this sounds like hot garbage

**Comment by ezrn** [comment]
> Why do u keep ruining legend sounds mykindos

nah youre off the goop with this one the new tooth sound is 100x better

**Comment by RealSoy** [comment]
okay whatever i dont really care about tooth sound, the constant lapis effect was the main thing i wanted to change, but ezrn you have to admit that the windblade and broadsword sound is awful

File: champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java [hunk 1]
```diff
@@ -109,8 +111,12 @@ public void doAlligatorsTooth() {
 
             VelocityData velocityData = new VelocityData(player.getLocation().getDirection(), velocityStrength, false, 0, 0.11, 1.0, true);
             UtilVelocity.velocity(player, null, velocityData);
-            player.getWorld().playEffect(player.getLocation(), Effect.STEP_SOUND, Material.LAPIS_BLOCK);
-            player.getWorld().playSound(player.getLocation(), Sound.ENTITY_FISH_SWIM, 0.8F, 1.5F);
+            
+            if (++soundUpdateCounter % 3 == 0)
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: soundUpdateCounter is "global", but is being used as if it is local to players.

File: champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java [hunk 2]
```diff
@@ -43,6 +43,8 @@ public class AlligatorsTooth extends ChannelWeapon implements InteractWeapon, Le
     private double bonusDamage;
     private double velocityStrength;
 
+    private int soundUpdateCounter;
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:46]: Probably should be a HashMap<UUID, Int>.

**Review by StudentAlleg — CHANGES_REQUESTED** [review]

File: champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java (hunk 1 above)
Comment (RealSoy) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: tbh idk what that means i just copied the code from mineplex, ill try my best though!

Comment (ezrn) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: just make a ticket about the effect playing too frequently and someone else will do it

Comment (Polisvh) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: Im telling you guiz listen to realsoy like srsly  windblade sound is terrible 
Idc about tooths sounds but actuallt lapis particles are hella annoying - barely can see anything while diving down the pool

Just turn it back to what it was on mineplex whats the deal

Comment (RealSoy) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: @Polisvh ong mineplex was so good at making sounds

Comment (Polisvh) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: @RealSoy yes I agree and these nubs dont see it. I will miss scepter sound forever </3

Comment (RealSoy) [reply champions/src/main/java/me/mykindos/betterpvp/champions/weapons/impl/legendaries/AlligatorsTooth.java:115]: new scepter is just stupid in general

**Pull request closed without merging**